web/src/routes/vote/referendum-ballot.svelte

```diff
@@ -0,0 +1,119 @@
+<script lang="ts">
+	import BallotEntry from "./ballot-entry.svelte";
+	import { type BallotEntry as BallotEntryT, fetching, user } from "../../store";
+	import List from "../../lib/list.svelte";
+	import Panel from "../../lib/panel.svelte";
+	import { createEventDispatcher } from "svelte";
+	
+	const options = [{
+		label: "For",
+		symbol: "thumb_up",
+		value: 1
+	}, {
+		label: "Against",
+		symbol: "thumb_down",
+		value: 2
+	}, {
+		label: "Abstain",
+		symbol: "back_hand",
+		value: 0
+	}] as { label: string, symbol: string, value: number }[]
```

Comment:
Remove typecast. TS should be able to figure out the narrowest type if marked with `as const` instead.